webrtc-stats.html

```diff
@@ -4223,6 +4226,33 @@ <h2>
                   no available memory. Calculated as defined in [[!RFC3550]] section 6.4.1.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>requestBytesSent</code></dfn> of type <span class=
+                "idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of bytes sent for connectivity checks.
```

Comment:
Does this include the headers or only the payload? We should be consistent across the dictionaries that the measurement of bytesSent and bytesReceived.

Reply:
The definition of "bytesSent" should be clear about whether ICE checks are counted or not. At the moment, I think it's not clear.
(I think they should not be included in "bytesSent", because they are overhead seen from the user. But the most important thing is that it is clear.)